### PR TITLE
[z-wave] Add Philio PAN08

### PR DESCRIPTION
**Pull request by jediperry**
Add the Philio 'Relay Insert for Blind Control' PAN08 to the database.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.zwave/database/philio/pan08.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+			<Minimum>1</Minimum>
+			<Maximum>10000</Maximum>
+			<WriteOnly>false</WriteOnly>
+			<Default>10000</Default>
+			<Size>2</Size>
+			<Label lang="en">Threshold of KWh for Load Caution</Label>
+		</Parameter>
+	</Configuration>
+	<Associations>
+		<Group>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/philio/pan08.xml]: Is this group meant to be linked to the controller or not?

Comment (jediperry) [inline bundles/binding/org.openhab.binding.zwave/database/philio/pan08.xml]: Actually, I'm not sure what this group is for. I left it in as it is listed in the manual and the related PAN06 has them.

**Comment by cdjackson** [comment]
It just looks like this is the only group. If it’s meant to be linked to the controller then we should set the configuration appropriately so that the device is configured correctly.  If not, then it’s fine, but I assume you have one of these (??) so you will need to work it out :)

**Comment by jediperry** [comment]
I do have one yes, and it works fine without making any association. Maybe it's for the overload alarm, but it's undocumented, so I don't think it's worth worrying about.

**Comment by cdjackson** [comment]
If it’s undocumented, then I would remove it - are you sure it exists? If you do a request to set it, does it work, or does it cause errors? The problem with having configuration or association groups that don’t exist in the device is they will cause timeouts, and the device to go DEAD.

**Comment by jediperry** [comment]
I mean the purpose is undocumented. The manual only says:
Association Groups:
1 (max. nodes in group: 1)

I tried an association with the controller earlier and it seems to accept it. I assume errors would show up in the debug log.
```
2015-09-13 19:52:02.372 [DEBUG] [ApplicationCommandMessageClass:38  ]- NODE 19: Application Command Request (ALIVE:DONE)
2015-09-13 19:52:02.373 [DEBUG] [ApplicationCommandMessageClass:56  ]- NODE 19: Incoming command class ASSOCIATION
2015-09-13 19:52:02.373 [DEBUG] [c.ZWaveAssociationCommandClass:96  ]- NODE 19: Received Association Request
2015-09-13 19:52:02.374 [DEBUG] [c.ZWaveAssociationCommandClass:156 ]- NODE 19: association group 1 has max associations 1
2015-09-13 19:52:02.374 [DEBUG] [c.ZWaveAssociationCommandClass:165 ]- NODE 19: association group 1 includes the following nodes:
2015-09-13 19:52:02.375 [DEBUG] [c.ZWaveAssociationCommandClass:170 ]- Node 1
2015-09-13 19:52:02.375 [DEBUG] [b.z.i.protocol.ZWaveController:595 ]- Notifying event listeners: ZWaveAssociationEvent
2015-09-13 19:52:02.376 [DEBUG] [.z.internal.ZWaveActiveBinding:433 ]- ZwaveIncomingEvent
2015-09-13 19:52:02.390 [DEBUG] [.b.z.i.p.i.ZWaveNodeSerializer:107 ]- NODE 19: Serializing to file etc/zwave/node19.xml
2015-09-13 19:52:02.408 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:63  ]- Sent message Message: class = SendData (0x13), type = Request (0x00), payload = 13 03 85 02 01
2015-09-13 19:52:02.409 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:64  ]- Recv message Message: class = ApplicationCommandHandler (0x04), type = Request (0x00), payload = 00 13 06 85 03 01 01 00 01
2015-09-13 19:52:02.410 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:65  ]- Checking transaction complete: class=ApplicationCommandHandler, expected=ApplicationCommandHandler, cancelled=false
2015-09-13 19:52:02.410 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:68  ]-          transaction complete!
2015-09-13 19:52:02.410 [DEBUG] [b.z.i.protocol.ZWaveController:595 ]- Notifying event listeners: ZWaveTransactionCompletedEvent
```

**Comment by cdjackson** [comment]
Ok - thanks. Looks fine then :)

**Pull request merged**